src/app/games/squarz/SquarzTutorial.ts

```diff
@@ -85,7 +85,7 @@ export class SquarzTutorial extends Tutorial {
                 }
             },
             TutorialStepMessage.CONGRATULATIONS(),
-        ).withPreviousMove(SquarzMove.from(new Coord(1, 1), new Coord(3, 3)).get()),
```

Comment:
NOOOOOOON JE REFUSE !

Reply:
Ps: réparer 4/5 au passage haha
![image](https://github.com/EveryBoard/EveryBoard/assets/41200522/dc4b43b3-96de-49f5-961d-bb417c8837b9)

Reply:
"it is called a capture. \n You are playing Light, try to do a capture" ?